variables.tf

```diff
@@ -148,7 +148,7 @@ variable "gcp_worker_instance_type" {
 
 variable "gcp_zone" {
   type        = string
-  default     = "us-central1-a"
+  default     = "us-east4-a"
```

Comment:
More of nit (doesn't seem like a big deal to change from one GCP zone to another), but IIRC we changed this zone because of L2 stuff, to make sure we created GCP and Equinix stuff in the right places to wire up the interconnection successfully